pkg/scheduler/framework/plugins/helper/spread_test.go

```diff
@@ -99,3 +101,159 @@ func TestGetPodServices(t *testing.T) {
 		})
 	}
 }
+
+func TestDefaultSelector(t *testing.T) {
+	const (
+		namespace                 = "test"
+		podName                   = "test-pod"
+		serviceName               = "test-service"
+		replicaSetName            = "test-replicaset"
+		replicationControllerName = "test-replicationcontroller"
+		statefulSetName           = "test-statefulset"
+
+		podLabelKey = "podLabelKey"
+		podLabelVal = "podLabelVal"
+
+		replicaSetLabelKey = "replicaSetLabelKey"
+		replicaSetLabelVal = "replicaSetLabelVal"
+
+		replicationLabelKey = "replicationLabelKey"
+		replicationLabelVal = "replicationLabelVal"
+
+		statefulSetLabelKey = "statefulSetLabelKey"
+		statefulSetLabelVal = "statefulSetLabelVal"
+	)
+
+	fakeInformerFactory := informers.NewSharedInformerFactory(&fake.Clientset{}, 0*time.Second)
+
+	// Create fake service
+	service := &v1.Service{
```

Comment:
Why the initialization is different with other objects, what if service not exists.

Reply:
I added  `addFakeService func() error` method and also injected when running test case